src/service/locator.cpp

```diff
@@ -524,7 +555,7 @@ locator_t::on_refresh(const std::vector<std::string>& groups) {
     const auto storage = api::storage(m_context, "core");
     const auto updated = storage->find("groups", std::vector<std::string>({"group", "active"}));
 
-    m_rgs.unsafe() = boost::accumulate(groups, *m_rgs.synchronize(),
+    routing.groups.unsafe() = boost::accumulate(groups, *routing.groups.synchronize(),
```

Comment:
Again whatever.

Also this change partially conflicts with #246, I'll rebase if necessary.